qpc/scan/job.py

```diff
@@ -72,11 +72,11 @@ def _build_req_params(self):
             found, scan_object_id = _get_scan_object_id(self.parser,
```

Comment:
maybe change it in the utility instead?

Reply:
It is already being cast as a string in the utility. Turns out only line 79 needs to be cast as a string.